hyperspy/io_plugins/README

```diff
@@ -13,6 +13,7 @@ All the read/write plugins must provide a python file containing:
         # Recognised file extension
         file_extensions = <Tuple of string>
         default_extension = <Int>	# Index of the extension that will be used by default
+        auto_convert_units = <Bool> # Convert automatically the units
```

Comment:
As above, I think that it'll be better not to hardwire this option in the reader.